### PR TITLE
#955 Improve documentation by fixing typos

### PR DESCRIPTION
**Pull request by snowteamer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ We are continually improving the user experience for everyone, and applying the
 
 #### Basic workflow
 
-Group Income is an application based on [Vue.js](https://vuejs.org/). Here's how you can get started the development:
+Group Income is an application based on [Vue.js](https://vuejs.org/). Here's how you can get started with the development:
```
Comment (taoeffect) [inline README.md]: Just `with development`

File: docs/Style-Guide.md [hunk 2]
```diff
@@ -102,16 +102,16 @@ When writing E2E tests, follow [Cypress Best Practices](https://docs.cypress.io/
...
-When the user performs an action (ex: submiting a form), it's expected to show them some type of feedback, on success or failure. There are 3 types of messages we want to display to the user:
+When the user performs an action (ex: submitting a form), it's expected to show them some type of feedback, on success or failure. There are three types of messages we want to display to the user:
 
 #### Type 1 - while filling a form
 
 These messages (mostly errors) are displayed whenever the user makes a mistake when filling out a form input. Examples:
 
 > A {fieldname} is required. —— Ex: A password is required.  
-> A {fieldname} cannot {action}. —— Ex: A username cannot contain spaces.  
+> A {fieldname} cannot {action}. —— Ex: An username cannot contain spaces.
```
Comment (taoeffect) [inline docs/Style-Guide.md]: Was correct as it was before

File: docs/Style-Guide.md [hunk 3]
```diff
@@ -174,7 +174,7 @@ submit () {
 
 #### Type 3 - App Error
 
-This banner informs the user that there's something going on with the app itself, or with something that is not related with a user action. Example:
+This banner informs the user that there's something going on with the app itself, or with something that is not related with an user action. Example:
```
Comment (taoeffect) [inline docs/Style-Guide.md]: Was correct as it was before

File: docs/Troubleshooting.md [hunk 4]
```diff
@@ -17,9 +17,9 @@ Try:
 
 *NOTE: this should no longer be an issue, but we're leaving this here just in case.*
 
-This can happen when you re-run grunt dev and then can no longer sign up a user because the database on the backend has been wiped, but the database in the browser's frontend still contains contracts that the frontend will try to re-subscribe to (that no longer exist on the backend).
+This can happen when you re-run `grunt dev` and then can no longer sign up an user because the database on the backend has been wiped, but the database in the browser still contains contracts that the frontend will try to re-subscribe to (although they no longer exist on the backend).
```
Comment (taoeffect) [inline docs/Troubleshooting.md]: `an user` => `a user`

**Review by taoeffect — CHANGES_REQUESTED** [review]
Excellent, amazing, fantastic work @snowteamer! Thank you for this very thorough contribution. I am impressed, you read all of our documentation! 😱 

Just a few minor issues noticed, think you can update the PR with the changes requested?

**Comment by taoeffect** [comment]
We'll merge this PR regardless since it has a lot of fantastic fixes, but I'll wait a little while in case you'd like to review the feedback and fix or respond to it.

File: docs/Style-Guide.md (hunk 2 above)
Comment (snowteamer) [reply docs/Style-Guide.md]: After some research, I have to agree with you on this point. So today I learned something new about English elision rules

**Comment by snowteamer** [comment]
Thank you for the kind words!

> Just a few minor issues noticed, think you can update the PR with the changes requested?

Sure, it will be a good way for me to remember that spelling rule.

**Comment by snowteamer** [comment]
And yes, I've read all the documentation. My favorite part was about the pox on modern web development, I found it well-explained and quite relaxing to read. I'm also planning to read the blog, starting with the article about voting systems. That's just what I generally do when I think a topic or project is really interesting.

**Review by taoeffect — APPROVED** [review]
Excellent work! Thank you for the PR!

**Pull request merged**